### PR TITLE
NurseConnect add metrics

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr please review.  The first block/list of commits are due to merge-in with #73

**Comment by NicolaasDieBaas** [comment]
@gsvr please review

File: test/ussd_nurse.test.js [hunk 1]
```diff
@@ -510,6 +514,11 @@ describe("app", function() {
                         state: 'state_end_reg',
                         reply: "Thank you. Weekly NurseConnect messages will now be sent to this number."
                     })
+                    .check(function(api) {
+                        var metrics = api.metrics.stores.test_metric_store;
+                        assert.deepEqual(metrics['test.ussd_nurse.self.registration_started'].values, [1]);
```
Comment (gsvr) [inline test/ussd_nurse.test.js]: just change this to `test_ussd_nurse.registrations_started` - note the plural, and that I don't want to track the `self` and `other` seperately.

**Review by gsvr — CHANGES_REQUESTED** [review]
Approved after a minor change

**Comment by NicolaasDieBaas** [comment]
resolving https://github.com/praekeltfoundation/ndoh-jsbox/issues/79 inside this issue

**Pull request merged**